src/app/organizations/registerOrganization/register-organization.component.ts

```diff
@@ -40,7 +40,7 @@ export class RegisterOrganizationComponent implements OnInit, OnDestroy {
   public TagEditorMode = TagEditorMode;
   constructor(
     private registerOrganizationService: RegisterOrganizationService,
-    private formsManager: AkitaNgFormsManager<FormsState>, @Inject(MAT_DIALOG_DATA) public data: any
+    private formsManager: AkitaNgFormsManager<FormsState>, @Inject(MAT_DIALOG_DATA) public data: any,
```

Comment:
I think you should revert this; not relevant to the change

Reply:
yes, forgot to remove comma